src/lib/datasource/abstract.coffee

```diff
@@ -16,6 +20,33 @@ class GridDatasourceAbstract
     return this
 
   ###
+   * Get/Set number of results returned from a query
+   * @param  {number} limit
+   * @return {this|number}
+  ###
+  limit: ( limit ) ->
+    return @_limit if typeof limit is 'undefined'
+    @_limit = limit
+    return this
+
+  ###
+   * Get/Set number of items to skip
+   * @param  {number} skip
+   * @return {this|number}
+  ###
+  skip: ( skip ) ->
```

Comment:
It depends: http://sailsjs.org/documentation/concepts/models-and-orm/query-language#?skip